modules/compliance-tailored-profiles.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-tailor.adoc
+
+[id="compliance-tailored-profiles_{context}"]
+= Using tailored profiles
+While the TailoredProfile CR enables the most common tailoring operations, the XCCDF standard allows even more flexibility in tailoring OpenSCAP profiles. In addition, if your organization has been using OpenScap previously, you may have an existing XCCDF tailoring file and can reuse it.
```

Comment:
`TailoredProfile` custom resource (CR)

if your organization previously used OpenScap, you might

Reply:
Extensible Configuration Checklist Description Format (XCCDF)